README.md

```diff
@@ -1,36 +1,32 @@
-Force
-=====
+# Force
 
 [Force](https://github.com/artsy/force) is the Artsy website, [www.artsy.net](https://www.artsy.net).
 
 Are you an Engineer? Don't know what Artsy is? Check out [this overview](https://github.com/artsy/meta/blob/master/meta/what_is_artsy.md) and [more](https://github.com/artsy/meta/blob/master/README.md).
 
 Want to know more about Artsy tech? Read the [Artsy Engineering Blog](http://artsy.github.io).
 
-Meta
-----
+## Meta
 
-* __State:__ production
-* __Production:__ [https://www.artsy.net](https://www.artsy.net) | [Heroku](https://dashboard.heroku.com/apps/force-production/resources)
-* __Staging:__ [https://staging.artsy.net](https://staging.artsy.net) | [Heroku](https://dashboard.heroku.com/apps/force-staging/resources)
-* __Github:__ [https://github.com/artsy/force](https://github.com/artsy/force)
-* __CI/Deploys:__ [CircleCi](https://circleci.com/gh/artsy/force); merged PRs to `artsy/force#master` are automatically deployed to staging; PRs from `staging` to `release` are automatically deployed to production. [Start a deploy...](https://github.com/artsy/force/compare/release...staging?expand=1)
-* __BrowserStack:__ For testing applications cross-browser use [BrowserStack](https://browserstack.com). Credentials are located in 1Password.
-* __Point People:__ [@kanaabe](https://github.com/kanaabe), [@damassi](https://github.com/damassi), [@mzikherman](https://github.com/mzikherman)
+* **State:** production
+* **Production:** [https://www.artsy.net](https://www.artsy.net) | [K8s](https://kubernetes-staging.artsy.net/#!/service/default/force-web?namespace=default)
```

Comment:
Rather than linking to just the service / deployment objects, I would link to `https://kubernetes.artsy.net/#!/search?q=force&namespace=default` which shows everything for Force